build/apply-certs-yugabyte.sh

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
+
+set -eo pipefail
+
+if [ "$1" == "" ]; then
+  echo "usage: ./apply-certs.sh <CLUSTER_CONTEXT> <NAMESPACE>"
+  exit 1
+fi
+
+if [ "$2" == "" ]; then
+  echo "usage: ./apply-certs.sh <CLUSTER_CONTEXT> <NAMESPACE>"
+  exit 1
+fi
+
+echo '========================================================================='
+echo '= Note that errors below are acceptable as long as the terminal message ='
+echo '= is success.                                                           ='
+echo '========================================================================='
+
+set -e
+set -x
+
+# Paths to directories in which to store certificates and generated YAML files.
+CONTEXT="$1"
+DIR="$(pwd)"
+NAMESPACE="$2"
+
+# Replace characters breaking folder names
+WORKSPACE=$(echo "${CONTEXT}" | tr ':/' '_')
+CLIENTS_CERTS_DIR="$DIR/workspace-yugabyte/$WORKSPACE/client_certs_dir"
+MASTER_CERTS_DIR="$DIR/workspace-yugabyte/$WORKSPACE/master_certs_dir"
+TSERVER_CERTS_DIR="$DIR/workspace-yugabyte/$WORKSPACE/tserver_certs_dir"
+# CA_KEY_DIR="$DIR/workspace/$WORKSPACE/ca_key_dir"
+# CA_CRT_DIR="$DIR/workspace/$WORKSPACE/ca_certs_dir"
+JWT_PUBLIC_CERTS_DIR="$DIR/jwt-public-certs"
+
+# Delete previous secrets in case they have changed.
+kubectl create namespace "$NAMESPACE"  --context "$CONTEXT" || true
```

Comment:
nit: define a command `kubectl --context xx --namespace` to reuse? That will avoid forgetting to set one of the two parameters, which can have not nice impact